pkg/plugin/client.go

```diff
@@ -132,12 +133,12 @@ func (d *DataSetClient) DoFacetRequest(req FacetRequest) (int, error) {
 	}
 	defer resp.Body.Close()
 
-	responseBytes, err := io.ReadAll(resp.Body)
+	respBytes, err := io.ReadAll(resp.Body)
 	if err != nil {
 		log.DefaultLogger.Error("error reading response from DataSet", "err", err)
 		return 0, err
 	}
-	log.DefaultLogger.Info("Result of request to facet", "body", string(responseBytes))
+	log.DefaultLogger.Info("Result of request to facet", "body", string(respBytes))
```

Comment:
Wouldn't it be better to have lower log level? `Debug` kind of?

Reply:
Yes, I've made that change